docs/UserGuide.md

```diff
@@ -1139,23 +1139,39 @@ Customer and order data are saved in the hard disk automatically after any comma
 
 ## FAQ
 
+### Customers
+{: .no_toc}
```

Comment:
Is it intended that only the Customers section is not included in the toc?

Reply:
Thanks for pointing it out!